SignalServiceKit/src/Loki/API/FileServerAPI.swift

```diff
@@ -52,6 +53,51 @@ public final class FileServerAPI : DotNetAPI {
         }
     }
     
+    @objc(downloadProfilePicture:)
+    public static func objc_downloadProfilePicture(_ downloadURL: String) -> AnyPromise {
+        return AnyPromise.from(downloadAttachment(downloadURL))
+    }
+    
+    // MARK: Attachment Download
+    @objc(downloadAttachment:)
+    public static func objc_downloadAttachment(_ downloadURL: String) -> AnyPromise {
+        return AnyPromise.from(downloadAttachment(downloadURL))
+    }
+    
+    public static func downloadAttachment(_ downloadURL: String) -> Promise<Data> {
```

Comment:
This is now no longer strictly related to the file server so let's move it to `DotNetAPI`